src/components/MyPage.js

```diff
@@ -84,8 +84,9 @@ class MyPage extends Component {
 
   signOut = async () => {
     try {
-      await AsyncStorage.removeItem('token');
-      Actions.login();
+      UserUtil.signOut((result, error) => {
+        AsyncStorage.removeItem('token', () => { Actions.login(); });
```

Comment:
you should put await prior to this AsyncStorage.removeItem. Otherwise, there is no point of making this function async without await